static/app/actionCreators/integrations.tsx

```diff
@@ -109,6 +109,29 @@ export function cancelDeleteRepository(
   return promise;
 }
 
+/**
+ * Delete a repository by setting its status to hidden.
+ *
+ * @param {Object} client ApiClient
+ * @param {String} orgId Organization Slug
+ * @param {String} repositoryId Repository ID
```

Comment:
```suggestion
 * @param client ApiClient
 * @param orgId Organization Slug
 * @param repositoryId Repository ID
```
types conflict with the typescript types